landingAr.html

```diff
@@ -11,17 +11,18 @@
 </head>
 
 <body>
+  <div class="dropdown landing">
+    <button class="dropbtn" id="landingbtn">تغير لغة ↓</button>
+    <div class="landing-content">
+      <a href="landing.html">English</a>
+      <a href="landingAr.html">العربية</a>
```

Comment:
in Arabic files
a tag for arabic should be the first

and the same for english file
a tag for english should be the first

Reply:
Personally, I don't think that really matters

Reply:
It is :v: